### ISSUE TITLE
twentytenfive_comments(): aria-required true failed

### ISSUE THREAD

**Issue by sprungmarker** [richardshepherd/TwentyTenFive#10]
twentytenfive_comments()

$aria_req for email and author is not filled even when it is required

$aria_req = ( $req ? " aria-required='true'" : '' ); is working

Thanks ...
